cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -62,7 +66,7 @@ function(find_and_configure_raft)
               "RAFT_COMPILE_NN_LIBRARY ${PKG_USE_RAFT_NN}"
               "RAFT_COMPILE_DIST_LIBRARY ${PKG_USE_RAFT_DIST}"
               "RAFT_USE_FAISS_STATIC ${PKG_USE_FAISS_STATIC}"
-              "RAFT_NVTX ${NVTX}"
+              "RAFT_NVTX ${PKG_NVTX}"
```

Comment:
@robertmaynard, hoping you might have an idea of why this doesn't seem to be turning on the option in RAFT. @achirkin mentioned that the explicit `set(RAFT_NVTX ON)` is needed in order for the RAFT to pick up the option.

Reply:
Ah ha, that totally makes sense. Thanks for explaining!